datatypes/src/commonMain/kotlin/at/asitplus/crypto/ecmath/ECMath.kt

```diff
@@ -140,11 +141,11 @@ operator fun BigInteger.times(point: ECPoint): ECPoint {
     var o = point
     var sum = if (this.bitAt(0)) point else point.curve.IDENTITY
     /* double-and-add */
-    for (i in 1..this.bitLength()) {
+    for (i in 1L..<this.bitLength()) {
```

Comment:
off-by-one error; didn't cause any issues here, but i noticed it when writing strauss-shamir